crates/occt-sys/Cargo.toml

```diff
@@ -148,7 +148,7 @@ exclude = [
     # "OCCT/src/BRepCheck/*",
     # "OCCT/src/MAT2d/*",
     "OCCT/src/ExpToCasExe/*",
-    # "OCCT/src/STEPControl/*",
+    "OCCT/src/STEPControl/*",
```

Comment:
This might have implications on whether or not I can publish to crates.io, we're currently right at the 10MB crate limit. We'll see how it goes

Reply:
I had issues where the STEPControl.hxx headers were missing on windows when it was not included.